docs/devices/9GED21500-005.md

```diff
@@ -54,9 +54,9 @@ The unit of this value is `%`.
 Can be set by publishing to `zigbee2mqtt/FRIENDLY_NAME/set` with payload `{"pin_code": {"user": VALUE, "user_type": VALUE, "user_enabled": VALUE, "pin_code": VALUE}}`
 To read (`/get`) the value publish a message to topic `zigbee2mqtt/FRIENDLY_NAME/get` with payload `{"pin_code": ""}`.
 - `user` (numeric): User ID to set or clear the pincode for 
-- `user_type` (enum): Type of user, unrestricted: owner (default), (year|week)_day_schedule: user has ability to open lock based on specific time period, master: user has ability to both program and operate the door lock, non_access: user is recognized by the lock but does not have the ability to open the lock allowed values: `unrestricted`, `year_day_schedule`, `week_day_schedule`, `master`, `non_access`
+- `user_type` (enum): Type of user, unrestricted: owner (default), (year|week)_day_schedule: user has ability to open lock based on specific time period, master: user has ability to both program and operate the door lock, non_access: user is recognized by the lock but does not have the ability to open the lock allowed values: `unrestricted`, `year_day_schedule`, `week_day_schedule`, `master`, `non_access`. year_day_schedule and week_day_schedule are not documented and may not be supported by this lock.
 - `user_enabled` (binary): Whether the user is enabled/disabled allowed values: `true` or `false`
-- `pin_code` (numeric): Pincode to set, set pincode to null to clear 
+- `pin_code` (numeric): Pincode to set, set pincode to null to clear. Because this is a numeric and not a string, it's not possible to set pin codes via UI that start with a zero (e.g. 0999 will be sent to the lock as 999). As a workaround, you can bypass the Z2M UI and publish an MQTT message manually as above with the pin code passed as a string.
```

Comment:
Please add this to the `Notes BEGIN` section above, this part is automatically generated.

Reply:
I'm closing and will open a new pull request.